applications/FluidDynamicsApplication/custom_elements/vms_adjoint_element.h

```diff
@@ -101,27 +109,29 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) VMSAdjointElement: public Element {
                 MakeIndirectScalar(r_node, AUX_ADJOINT_FLUID_VECTOR_1_X, Step);
             rVector[index++] =
                 MakeIndirectScalar(r_node, AUX_ADJOINT_FLUID_VECTOR_1_Y, Step);
-            if (mpElement->GetGeometry().WorkingSpaceDimension() == 3)
-            {
+            if (mpElement->GetGeometry().WorkingSpaceDimension() == 3) {
                 rVector[index++] =
                     MakeIndirectScalar(r_node, AUX_ADJOINT_FLUID_VECTOR_1_Z, Step);
             }
             rVector[index] = IndirectScalar<double>{}; // pressure
         }
 
-        void GetFirstDerivativesVariables(std::vector<VariableData const*>& rVariables) const override
+        void GetFirstDerivativesVariables(
```

Comment:
Style guide says that one argument functions must be in one line.

Reply:
done :)